astroid/nodes/as_string.py

```diff
@@ -183,7 +183,7 @@ def visit_classdef(self, node):
             args.append("metaclass=" + node._metaclass.accept(self))
         args += [n.accept(self) for n in node.keywords]
         args = f"({', '.join(args)})" if args else ""
-        docs = self._docs_dedent(node.doc) if node.doc else ""
+        docs = self._docs_dedent(node.doc_node.value) if node.doc_node else ""
```

Comment:
Should `self._docs_dedent` accept `nodes.Const | None` instead?

Reply:
See https://github.com/PyCQA/astroid/pull/1448